.werft/jobs/build/public-api/jwt-signing-key.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
+  jwt-signing-key: and0LWtleS1mb3ItcHJldmlldy1lbnZzCg==
```

Comment:
It would make sense to use `certmanager` for this which can automatically issue these keys, so we don't need to define them

Reply:
Can we make use of `certmanager` in a different task? Then we can also optimize the PAT signing secret.

Reply:
PAT doesn't use a JWT - the resulting token would be too long.

Reply:
> Can we make use of certmanager in a different task?
Sure, tho I think manually injecting it is actually more work than using certmanager

Reply:
This is about a signing key. That's quite the same task that needs to be performed to sign and verify PATs.

Reply:
PATs do not use JWTs. They use HS-256 which is a symmetric signing algorithm (https://auth0.com/docs/get-started/applications/signing-algorithms), this is done because signatures which can be verified using a public-key are longer, and because the signature is part of the token, we couldn't use RS256.

That also means the PAT signging keys cannot be generated with certmanager, it just doesn't support this kind of key.